bin/generate_client

```diff
@@ -73,12 +75,15 @@ do
 
       given_out_dir=1
       if [ ${output_volume:0:2} == "./" ]; then
-        ouput_volume=${PWD}/${output_volume#*./}
+        output_volume=${PWD}/${output_volume#*./}
       elif [ ${output_volume:0:1} != "/" ]; then
         output_volume=${PWD}/${output_volume}
       fi
       shift
       ;;
+    -u|--update)
```

Comment:
Why did we need a new command line arg for this fix?

Reply:
Mainly [this bit](https://github.com/cyberark/conjur-openapi-spec/pull/195/files#diff-4db92db5652efb677efaed8d6edbdb44593d6502b2948925aab5c2a4a15808b0L102-L104). The output directory of the `bin/generate_client` call in `update-wiki.yml` was targeting the recently cloned wiki repo - I think the repo was being cloned, removed, and then the directory recreated instead of the files being replaced and pushed.

Reply:
I'll activate the GHA on this PR to confirm behavior before you approve.

Reply:
[Wiki update successful](https://github.com/cyberark/conjur-openapi-spec/actions/runs/932275890). Going to remove the PR trigger.